python/setup.py

```diff
@@ -250,14 +257,22 @@ def run(self):
         license='http://www.apache.org/licenses/LICENSE-2.0',
         # Don't forget to update python/docs/source/getting_started/install.rst
         # if you're updating the versions or dependencies.
-        install_requires=['py4j==0.10.9.2'],
+        install_requires=[
+            'py4j==0.10.9.2',
+            'pandas>=0.23.2',
+            'pyarrow>=0.10',
+            'numpy>=1.14,<1.20.0',
```

Comment:
I don't think we should install them by default but should be extras.

Reply:
They are moved to `extras_require`.

Does it mean `when users install pyspark, pandas_on_spark doesn't work by default`?